evap/contributor/views.py

```diff
@@ -227,6 +227,7 @@ def evaluation_edit(request, evaluation_id):
         "editable": True,
         "preview_html": preview_html,
         "questionnaires_with_answers_per_contributor": {},
+        "show_participant_cms_info": evaluation.exam_type is not None and evaluation.cms_id is not None,
```

Comment:
Can we keep this decision in the form? Maybe we could set something like `self.fields["particpants"].cms_disclaimer = _(...)` and pick this up in the view?